### DISCUSSION TITLE
LeadImages vs MainImage

### DISCUSSION THREAD

**Discussion by adgad** [Financial-Times/content-tree#21] [Team Posts]
From the perspective of the article page, `leadImages` and `mainImage` server very similar purposes (although the `mainImage` is also used in other contexts i.e. teasers)

 This is more noticeable now that `mainImage` can have multiple members, with the FT Edit square image.
 
 Should these be consolidated somehow?

**Comment by atanasdinov** [comment]
`leadImages` is an internal-only field in the Article, Live Blog Post & Live Blog Package schemas. Consolidating `leadImages` and `mainImage` only for internal clients seems like a rather risky plan and it poses an even greater risk when fixing older content.
